src/swap/components/Swap.tsx

```diff
@@ -50,10 +52,14 @@ export function Swap({
         )}
         data-testid="ockSwap_Container"
       >
-        <div className="mb-4">
-          <h3 className={text.title3} data-testid="ockSwap_Title">
+        <div className="mb-4 flex items-center justify-between">
+          <h3
+            className={cn(text.title3, 'text-inherit')}
+            data-testid="ockSwap_Title"
+          >
             {title}
           </h3>
+          {swapSettings}
```

Comment:
Let's add this later, in the next PR. Otherwise this will block the merging of this one.